cbits/ftoa.c

```diff
@@ -0,0 +1,536 @@
+// Copyright 2018 Ulf Adams
+//
+// The contents of this file may be used under the terms of the Apache License,
+// Version 2.0.
+//
+//    (See accompanying file LICENSE-Apache or copy at
+//     http://www.apache.org/licenses/LICENSE-2.0)
+//
+// Alternatively, the contents of this file may be used under the terms of
+// the Boost Software License, Version 1.0.
+//    (See accompanying file LICENSE-Boost or copy at
+//     https://www.boost.org/LICENSE_1_0.txt)
```

Comment:
Are either of the licenses acceptable for inclusion in `bytestring`

Reply:
Prefacing that I am not an expert in licenses but they seem to both be quite permissive. I believe the conditions for Apache are just that we include the license and note that copyright and modifications, which have been done.